README.md

```diff
@@ -60,6 +58,10 @@ This step isn't needed if you download a release instead.
 
 ## Changelog
 
+### 2.1.0
+
```

Comment:
There is one line break too many here (for consistency with other versions) @decodekult